docs/rfcs/1002-merging-process.md

```diff
@@ -286,56 +286,60 @@ bot). Take the time to read dependency changelog, blog posts, or similar. Keep
 in mind that this step could prevent (to a certain limit) a potential
 supply-chain attack where a fake update is released with malicious code.
 
-## How do we merge a Pull-Request
+## Merging a pull request
 
-When the PR was reviewed and accepted,
-at that moment you can merge it to the main branch.
+After the pull request has been reviewed and accepted, the author is allowed to
+merge it into the main branch.
 
-But we want to merge it in a specific way:
+The merge should be done in a specific way:
 
-- We want that your commit that you're adding appear on top of the git history (see [Why do we want a linear git history](#why-do-we-want-a-linear-git-history)).
-- We want to test one last time if your changes are compatible with the main branch.
+- The commits should appear on top of the git history (see [Why do we want a linear git history](#why-do-we-want-a-linear-git-history)).
+- It should be tested one last time if changes are compatible with the main
+  branch.
 
-For that we will use a merge queue (see [What is a merge queue](#what-is-a-merge-queue) that will rebase your commit onto the main branch.
+This is done by the [merge queue](#what-is-a-merge-queue) that will rebase the
+commits onto the main branch.
```

Comment:
```suggestion
    This is done by the [merge queue](#what-is-a-merge-queue) that will rebase the
    commits onto the main branch.
```

I think the paragraph should be a part of the last element `It should be tested one last time ...`